applications/structure/bar/application.h

```diff
@@ -387,7 +387,7 @@ class Application : public ApplicationBase<dim, Number>
     if(this->param.problem_type == ProblemType::QuasiStatic)
       quasistatic_solver = true;
 
-    if(boundary_type == "Dirichlet")
+    if(boundary_type == Dirichlet)
```

Comment:
Why is it just `Dirichlet` and not something like `BoundaryType::Dirichlet`?

Reply:
[fe22560](https://github.com/exadg/exadg/pull/382/commits/fe22560ec03fbd3134afc96a469110348093d320)